languages.toml

```diff
@@ -2092,3 +2092,18 @@ comment-token = "//"
 [[grammar]]
 name = "ponylang"
 source = { git = "https://github.com/mfelsche/tree-sitter-ponylang", rev = "ef66b151bc2604f431b5668fcec4747db4290e11" }
+
+[[language]]
+name = "dhall"
+scope = "source.dhall"
+injection-regex = "^dhall$"
```

Comment:
```suggestion
injection-regex = "dhall"
```

Reply:
I should update the docs, where the example for the injection regex uses anchors (https://github.com/helix-editor/helix/blob/master/book/src/languages.md?plain=1#L38). Is it ok to do it in this PR, or in another one?

Reply:
done, but i don't know how to update `book/generated`